aii-core/src/main/perl/Shellfe.pm

```diff
@@ -859,6 +895,27 @@ sub check_protected {
     return %hash;
 }
 
+sub change_dhcp 
+{
+    my ($self, $method, %nodes) = @_;
+    $self->debug(5, "logfile:", $self->option('logfile'), " dhcpcfg:", $self->option('dhcpcfg'));
+    my $dhcpmgr = AII::DHCP->new('script',
+        "--logfile=".$self->option('logfile'), 
+        "--cfgfile=".$self->option('dhcpcfg'), 
```

Comment:
@stdweird thanks for the clarification. I now understand that `AII::DHCP` initialize a different CAF::Application. I found this not very clean but I think we can open an issue for the next release (about unifying config files and have only shared by `aii-shellfe` and all its components).